fabtests/functional/av_xfer.c

```diff
@@ -237,6 +237,7 @@ int main(int argc, char **argv)
 		      FI_TAGGED : FI_MSG;
 	hints->mode = FI_CONTEXT;
 	hints->domain_attr->mr_mode = opts.mr_mode;
+	hints->addr_format = opts.address_format;
```

Comment:
The above 2 lines are either in every fabtest, or nearly every test.  We need a more maintainable way of handling this.  But I'm okay dealing with that later.